.github/workflows/spm.yml

```diff
@@ -48,10 +48,13 @@ jobs:
       matrix:
         target: [tvOS, macOS, catalyst]
         # Full set of Firebase-Package tests only run on iOS because of Analytics.
```

Comment:
Nit: comment should go above now? or not relevant anymore?

Reply:
Still relevant